tests/python/tddft/TDDFT.py.dat

```diff
@@ -0,0 +1,320 @@
+   @DF-RHF iter  17:  -113.24480600149194   -6.71760e-09   1.00469e-06 DIIS
+   @DF-RHF iter  18:  -113.24480600683268   -5.34074e-09   6.85187e-07 DIIS
+  Energy converged.
+
+
+  ==> Post-Iterations <==
+
+    Orbital Energies [Eh]
+    ---------------------
+
+    Doubly Occupied:                                                      
+
+       1A    -20.648574     2A    -11.492685     3A     -1.201791  
+       4A     -0.792647     5A     -0.518971     6A     -0.517564  
+       7A     -0.406688     8A     -0.385868  
+
+    Virtual:                                                              
+
+       9A     -0.041555    10A      0.012424    11A      0.048190  
+      12A      0.096944    13A      0.726290    14A      0.797017  
+      15A      0.804759    16A      0.840245    17A      1.020597  
+      18A      1.028553    19A      1.171889    20A      1.187544  
+      21A      1.231886    22A      1.498962  
+
+    Final Occupation by Irrep:
+              A 
+    DOCC [     8 ]
+
+  @DF-RHF Final Energy:  -113.24480600683268
+
+   => Energetics <=
+
+    Nuclear Repulsion Energy =             16.4911059382062462
+    One-Electron Energy =                -187.5422568777643733
+    Two-Electron Energy =                  57.8063449327254375
+    Total Energy =                       -113.2448060068326754
+
+Computation Completed
+
+
+Properties will be evaluated at   0.000000,   0.000000,   0.000000 [a0]
+
+Properties computed using the SCF density matrix
+
+  Nuclear Dipole Moment: [e a0]
+     X:     0.0000      Y:     0.0000      Z:    -4.0980
+
+  Electronic Dipole Moment: [e a0]
+     X:     0.0001      Y:     0.0001      Z:     3.2554
+
+  Dipole Moment: [e a0]
+     X:     0.0001      Y:     0.0001      Z:    -0.8425     Total:     0.8425
+
+  Dipole Moment: [D]
+     X:     0.0002      Y:     0.0001      Z:    -2.1415     Total:     2.1415
+
+
+*** tstop() called on Daniels-MacBook-Pro.local at Sat Nov 10 09:07:45 2018
+Module time:
+	user time   =       0.49 seconds =       0.01 minutes
+	system time =       0.20 seconds =       0.00 minutes
+	total time  =          2 seconds =       0.03 minutes
+Total time:
+	user time   =       0.49 seconds =       0.01 minutes
+	system time =       0.20 seconds =       0.00 minutes
+	total time  =          2 seconds =       0.03 minutes
+
+Traceback (most recent call last):
+  File "../../../psi4/run_psi4.py", line 272, in <module>
+    exec(content)
+  File "<string>", line 38, in <module>
+  File "/Users/daniel/Gits/psi4ds/psi4/driver/p4util/solvers.py", line 408, in davidson_solver
+    print("    Root {:>5}: |Imag[A_w[{}]]| > 1.0E-10!".format(i))
+
+IndexError: tuple index out of range
+
+Printing out the relevant lines from the Psithon --> Python processed input file:
+        return prod.H2H1_product(vector)
+    def precon(resid, i, A_w):
+        return resid
+    nvecs = 5
+    guess = np.ones((prod.narot, nvecs))
+--> davidson_solver(func, precon, guess, no_eigs=nvecs, e_conv=1.e-4)
```

Comment:
Does this file need to be updated? Or should it be removed?

Reply:
Completely forgot about this stub that was added as the starting point. I will remove it.

Reply:
On the topic of removing stuff. There were some files/lines of code in `libfock` related to TD stuff that were either incomplete implementations or are/were unused. Can you remove those too?

Reply:
Im going to remove the TDDFT test as that is implemented/replaced by the tests I added with this PR. However the stability stub that is in place I will leave as it will be useful to whomever decides to work on that down the road